nservicebus/msmq/sender-side-distribution-with-distributor.include.md

```diff
@@ -0,0 +1 @@
+INFO: When using-sender side distribution in combination with the distributor, delayed retries and delayed deliveries will be routed to the distributor even when the messages were initially sent directly to the endpoint instance, bypassing the distributor (e.g. using sender-side distribution).
```

Comment:
This is somewhat confusing. Will the retry messages be sent to the distributor or the instance? first part says that it will be routed to the distributor, second part says that it will be bypassing the distributor. 

Do you mean the following?
INFO: Using Sender-Side distribution feature in combination with the Distributor will affect the routing of [delayed retries](/nservicebus/recoverability/configure-delayed-retries.md) and [delayed delivery](/nservicebus/messaging/delayed-delivery.md) of messages. These messages will be routed directly to the Distributor instead of the endpoint instance even though these messages were originally sent to the endpoint instance.